scout/server/blueprints/variants/controllers.py

```diff
@@ -49,28 +50,46 @@ def variants(store, institute_obj, case_obj, variants_query, page=1, per_page=50
         overlapping_svs = [sv for sv in store.overlapping(variant_obj)]
         variant_obj['overlapping'] = overlapping_svs or None
 
+        # show previous classifications for research variants,
+        is_research = False
+        if variant_obj['variant_type'] == 'research':
+            is_research = True
+
         # Get all previous ACMG evalautions of the variant
         evaluations = []
         for evaluation_obj in store.get_evaluations(variant_obj):
             classification = evaluation_obj['classification']
             # Only show pathogenic/likely pathogenic from other cases on variants page
             if evaluation_obj['case_id'] == case_obj['_id']:
-                continue
+                if not is_research:
+                    continue
             if not classification in ['pathogenic', 'likely_pathogenic']:
-                continue
+                if not is_research:
+                    continue
             # Convert the classification int to readable string
             evaluation_obj['classification'] = ACMG_COMPLETE_MAP.get(classification)
             evaluations.append(evaluation_obj)
         variant_obj['evaluations'] = evaluations
 
+        # Show previous classifications from the clinical side for research
+        if is_research:
+            # get variant by simple_id. That will really just return the first variant found -
+            # but mostly that would be clinical.. Its a start.
+            clinical_var_obj = store.variant(case_id=case_obj['_id'],
```

Comment:
What about querying all events by simple_id ('subject')? That would give an indication of the actions associated to the specific clinical variants, and from there you could collect their _id or document_id

Reply:
Oh, sorry about the comment - I gave the adapter variant an extra argument. It is better this way: before it was a lottery for anyone which type of variant would be returned if both clinical and research existed.